components/test/testPolyfills.ts

```diff
@@ -14,7 +14,7 @@ window.location = {
   search: '?testTorrentId'
 } as any
 
-global.decodeURIComponent = () => 'test'
+global.decodeURIComponent = (url) => url
```

Comment:
cc @fallaciousreasoning @petemill : `decodeURIComponent` was mocked to always return 'test' 🤔, but not sure why it was like this. Does this change look okay?

(It's interesting that it's not mocking encodeURICompoenent() on the other hand. If the mock was filling non-existing decodeURIComponent in global, why not encodeURIComponent()?)

https://github.com/brave/brave-core/commit/f959d2e585843bf6a1d53a5353d67279a3aa75bc

Reply:
I wonder if we could just remove the mock entirely - tbh, I'm not sure what the point of this is (and if a specific test needs this, then that specific test should set this up, and put things back to normal when it's done. I don't really think we should be globally overriding this....

Reply:
Yeah, CI looks okay without it.